backend/mypy.ini

```diff
@@ -0,0 +1,9 @@
+[mypy]
+plugins =
+    mypy_django_plugin.main,
+    mypy_drf_plugin.main
+ignore_missing_imports = True
+strict = True
+
+[mypy.plugins.django-stubs]
+django_settings_module = backend.settings
```

Comment:
We need a blank line at the end of the file :)

Reply:
Same here :). Btw i think i can implement pre-commit if you are intrested. I started using it and it has a end-of-line fixer hook that would take care of this and it also can run black to get rid of the issue  above before submitting a PR

Reply:
For end of line formatting I think that would make sense, @to-sta! Thanks for the offer to implement!